source/references/components/indicator.md

```diff
@@ -10,23 +10,24 @@ version: 2.1
 
 ### Summary
 
-This component must be used as a subcomponent of a [`slider`](./slider.html) component.
+The `<indicator>` component usually used to show an indicator effect, it must be used as a subcomponent of a [`slider`](./slider.html) component.
```

Comment:
The `indicator` component is to present the indicator in a `slider`, and it must be used as a child component for a dedicated `slider` component.